t/01_basic.t

```diff
@@ -1,13 +1,67 @@
-use Mojo::Base -strict;
-use Test::Mojo;
+## no critic
+package MyPgRest;
+use Mojolicious::Lite -strict;
+use Mojo::JSON qw(decode_json);
+
+my $api = <<EOAPI;
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Sample API",
+    "description": "API description in Markdown.",
+    "version": "1.0.0"
+  },
+  "host": "localhost:3000",
+  "basePath": "/v1",
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/users": {
+      "get": {
+        "summary": "Returns a list of users.",
+        "description": "Optional extended description in Markdown.",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    }
+  }
+}
+EOAPI
+
+get '/' => sub { my $c = shift; $c->render( json => decode_json($api) ) };
+
+# Plugin must be initialized at end of application because we
+# need to set the '/' (root route) before
+plugin PgREST => { openApi => app->ua->server->url };
+
+1;
+
+package main;
+## use critic
 use Test::More;
+use Test::Mojo;
+
+use Mojolicious -signatures;
```

Comment:
Rethinking, I don't need this, because this was written to wrap the mock application